README.md

```diff
@@ -1,4 +1,52 @@
-docker-base-images
-==================
+# Apiary's base Docker images
 
-Base docker images for Apiary applications
+This repository helps to keep Apiary environment, services and libraries consistent and in sync. 
+
+Each directory have area of responsibility and the libraries should derive from the respective image.
+
+As a lot of applications/libraries are deployed on Heroku, images are usually derived from the Ubuntu version from the latest Heroku stack. Libraries use what's the minimal stack available.
+
+## Naming
+
+Base development images uses `base-dev-` prefix.
+
+Base deployment/runtime images uses `base-` prefix.
+
+## Versioning
+
+We keep `latest` tag for testing and that is updated at-will. Otherwise, semver is used:
```

Comment:
If `semver` is used, use commonly recognized terms... this is the first time in my life I read about `arch` version. Semver uses `MAJOR.MINOR.PATCH`.

Also, semver could be hyperlinked to the website.